### PR TITLE
Update Ingress template for new apiVersions

### PR DESCRIPTION
**Pull request by briantopping**

### CODE REVIEW

File: charts/frigate/Chart.yaml [hunk 1]
```diff
@@ -11,6 +11,10 @@ home: https://github.com/blakeblackshear/blakeshome-charts/tree/master/charts/fr
 icon: https://github.com/blakeblackshear/frigate/blob/master/docs/static/img/frigate.png
 sources:
   - https://github.com/blakeblackshear/frigate
+dependencies:
+- name: common
+  version: 3.0.0
+  repository: https://k8s-at-home.com/charts/
```
Comment (onedr0p) [inline charts/frigate/Chart.yaml:17]: I am not sure if we want to have the dependencies on this library chart. Might be better to copy over any logic/code needed to make the ingress work.

Comment (briantopping) [reply charts/frigate/Chart.yaml:17]: Hi thanks for looking at the PR :)

I figured there was some reasoning behind the missing content in the `helm package` output. This solves it, but it sounds like you have more experience than I do with it. What are the downsides of including this?

Comment (skateman) [reply charts/frigate/Chart.yaml:17]: If it's not needed for ingress, I think it should not be included in the context of this PR regardless of what this does.

Comment (ArturZurawski) [reply charts/frigate/Chart.yaml:17]: @briantopping could we have this removed please? Merging this would be nice :)

Comment (briantopping) [reply charts/frigate/Chart.yaml]: Removed, apologies for delay.

**Comment by onedr0p** [comment]
Sorry there was a PR already merged to address this.

**Pull request closed without merging**